components/Contact/ContactInfo.vue

```diff
@@ -10,23 +10,23 @@
       </el-select>
     </el-form-item>
     <el-form-item v-if="firstName.visible" :required="firstName.required" :label="firstName.display">
-      <el-input v-model="firstName.value" @change="fieldChange('firstName')" :maxlength="firstName.maxLength" :placeholder="firstName.placeholder" :disabled="firstName.disabled"></el-input>
+      <el-input v-model="firstName.value" @input="fieldinput('firstName')" :maxlength="firstName.maxLength" :placeholder="firstName.placeholder" :disabled="firstName.disabled"></el-input>
```

Comment:
Need to be more consistent with the casing. I expect it to be in camel case - fieldInput